main/src/ca/uwaterloo/flix/language/phase/Monomorpher.scala

```diff
@@ -114,18 +131,9 @@ object Monomorpher {
       * Performance Note: We are on a hot path. We take extra care to avoid redundant type objects.
```

Comment:
can't comment the correct location, but StrictSubstitution doc comment in "Notes on `s`:" should be updated